src/js/serial.js

```diff
@@ -231,23 +234,22 @@ const serial = {
 
                 const disconnectFn = (self.connectionType === 'serial') ? chrome.serial.disconnect : chrome.sockets.tcp.close;
                 disconnectFn(self.connectionId, function (result) {
-                    checkChromeRuntimeError();
-
+                    if (chrome.runtime.lastError) {
+                        console.log(chrome.runtime.lastError.message);
```

Comment:
What is the reason that we aren't using `checkChromeRuntimeError()` here?

Reply:
Maybe not needed anymore but I should not hurt. Needed the code to have less overhead to improve timing.